pkg/signature/kms/hashivault/client.go

```diff
@@ -200,9 +199,14 @@ func (h hashivaultClient) verify(sig, digest []byte, alg crypto.Hash) error {
 	client := h.client.Logical()
 	encodedSig := base64.StdEncoding.EncodeToString(sig)
 
+	vaultDataPrefix := os.Getenv("VAULT_KEY_PREFIX")
+	if vaultDataPrefix == "" {
+		vaultDataPrefix = "vault:v1:"
```

Comment:
Define this as a `const` above?

Reply:
Adjusted as requested. Thanks for the good feedback!